functions/trello/trelloOauth.ts

```diff
@@ -0,0 +1,140 @@
+import express, { Request, Response } from "express";
+import OAuth from "oauth";
+import url from "url";
+import fs from "fs";
+import { fetchAndProcessTrelloData } from "./index";
+
+const boardId = "";
+/**
+ * Initializes and configures the Express server.
+ */
+const app = express();
+app.use(express.static("public"));
+
+/**
+ * Starts the server and listens on port 3000.
+ */
+const server = app.listen(3000, () => {
```

Comment:
You must be using the export default else this will not work when hosting on nhost